### PR TITLE
feat: Make IPFS instance on CLI non-disposable

### PR DESCRIPTION
**Pull request by ukstv**
When run in CLI mode, IPFS instance is non-disposable.

Test procedure:
- run ceramic daemon,
- create a stream,
- stop ceramic daemon,
- run ceramic daemon,
- query state of the stream.

Reports the state just all right.

### CODE REVIEW

**Review by stbrody — APPROVED** [review]
🚀

File: packages/cli/src/build-ipfs-connection.util.ts [hunk 1]
```diff
@@ -58,6 +47,7 @@ async function createGoIPFS(overrideConfig: Partial<ipfsClient.Options> = {}): P
   const gatewayPort = 9011
   const defaultConfig = {
     ipld: { codecs: [dagJose] },
+    repo: '~/.ipfs',
```
Comment (stephhuynh18) [inline packages/cli/src/build-ipfs-connection.util.ts:50]: Is this the default repo we want to use? I don't mind either way but before it was `.jsipfs` and in `ipfs-daemon` its `.goipfs`

**Review by stephhuynh18 — APPROVED** [review]
One question tho

File: packages/cli/src/build-ipfs-connection.util.ts (hunk 1 above)
Comment (ukstv) [reply packages/cli/src/build-ipfs-connection.util.ts:50]: true. thanks!

**Pull request merged**